docker_test_tools/environment.py

```diff
@@ -254,15 +256,18 @@ def get_container_id(self, name):
         :param str name: container name as it appears in the docker compose file.
         """
         self.validate_service_name(name)
-        try:
-            output = subprocess.check_output(
-                ['docker-compose', '-f', self.compose_path, '-p', self.project_name, 'ps', '-q', name],
-                stderr=subprocess.STDOUT, env=self.environment_variables
-            )
-        except subprocess.CalledProcessError as error:
-            raise RuntimeError("Failed getting container %s id, reason: %s" % (name, error.output))
-
-        return utils.to_str(output)
+        filters = {
+            "label": "com.docker.compose.service={service}".format(service=name)
+        }
+        containers = self.docker_client.containers(filters=filters)
+        containers = [container for container in containers
+                      if 'com.docker.compose.project' in container['Labels'] and
+                      container['Labels']['com.docker.compose.project'] == self.project_name]
+        if not containers:
+            raise RuntimeError("No containers found for name %s and project %s" % (name, self.project_name))
+        if len(containers) > 1:
+            raise RuntimeError("More than one container were found for name %s and project %s" % (name, self.project_name))
```

Comment:
Instead of two validations you can use one
```
if len(containers) != 1:
    raise RuntimeError(...)
```

Reply:
please recommend on a message that will fit both cases, today the error message explains exactly what was wrong

Reply:
`Unexpected containeres number (%d) were found for name %s and project %s`